example/src/components/EtchRunes/index.tsx

```diff
@@ -0,0 +1,147 @@
+import { useMemo, useState } from 'react';
+import Wallet, { Address, AddressPurpose, BitcoinNetworkType } from 'sats-connect';
+
+type Props = {
+  network: BitcoinNetworkType;
+  addresses: Address[];
+};
+
+const EtchRunes = ({ addresses, network }: Props) => {
+  const [totalCost, setTotalCost] = useState<number>();
+  const [totalSize, setTotalSize] = useState<number>();
+  const [fundTxId, setFundTxId] = useState<string>('');
+  const [runeName, setRuneName] = useState<string>('');
+  const [feeRate, setFeeRate] = useState<string>('');
+  const [symbol, setSymbol] = useState<string>('');
+  const [preMine, setPreMine] = useState<string>('');
+  const [divisibility, setDivisibility] = useState<string>('');
+
+  const ordinalsAddress = useMemo(
+    () => addresses.find((a) => a.purpose === AddressPurpose.Ordinals)?.address || '',
+    [addresses]
+  );
+
+  const paymentAddress = useMemo(
+    () => addresses.find((a) => a.purpose === AddressPurpose.Payment)?.address || '',
+    [addresses]
+  );
+
+  const onClickEstimate = async () => {
+    const response = await Wallet.request('runes_estimateEtch', {
```

Comment:
We should add an amount & cap. Open mints without a cap are unmintable.

Reply:
what happens on the service if you submit an order with `isMintable=true` and no amount & cap?

Reply:
we are working on adding all validations of required parameters & valid combinations, we are running short on time on adding API unit tests. This uncapped openmint that creates unmintable runes is not even documented in runes spec, I found out about it from ord code.

The desired behavior is that API should reject creating unmintable runes, we need to add all scenarios this week.

Reply:
It will throw a 400

Reply:
That would be caught in the sats-connect method and would return an error result here, so maybe that's enough?